http_logs/index.json

```diff
@@ -12,38 +12,118 @@
     "properties": {
       "@timestamp": {
         "format": "strict_date_optional_time||epoch_second",
-        "type": "date"
+        "type": "date",
+        "doc_values": {{ timestamp_doc_values | default(true) | tojson }},
+        "index": {{ timestamp_index | default(true) | tojson }}
       },
-      "clientip": {
-        "type": "ip"
-      },
-      "message": {
-        "type": "keyword",
-        "index": false,
-        "doc_values": false
-      },
-      "request": {
-        "type": "text",
-        "fields": {
-          "raw": {
-            "ignore_above": 256,
-            "type": "keyword"
+      {%- if runtime_grok is defined %}
```

Comment:
This implements using my `grok` prototype. I'm 10000% sure we aren't going to keep this API, but it is something.